### YamlMime:UniversalReference
api_name: []
items:
- children:
  - azure.keyvault.models.error_py3.Error
  fullName: azure.keyvault.models.error_py3
  langs:
  - python
  module: azure.keyvault.models.error_py3
  name: error_py3
  source:
    id: error_py3
    path: azure-keyvault\azure\keyvault\models\error_py3.py
    remote:
      branch: master
      path: azure-keyvault\azure\keyvault\models\error_py3.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 0
  type: module
  uid: azure.keyvault.models.error_py3
references:
- fullName: azure.keyvault.models.error_py3.Error
  isExternal: false
  name: Error
  parent: azure.keyvault.models.error_py3
  uid: azure.keyvault.models.error_py3.Error
